Makefile

```diff
@@ -175,7 +175,9 @@ _verify_pa11y_testable:
 _run_pa11y:
 	echo $(CIRCLE_BRANCH)
 ifneq ($(CIRCLE_BRANCH),)
-	@export TEST_URL=http://${TEST_APP}.herokuapp.com; pa11y-ci;
+	@export TEST_URL=${TEST_APP}; \
```

Comment:
I want to add a comment here to explain, but it makes it error.

Makefile still scares me a bit